config.yml

```diff
@@ -16,37 +16,38 @@ params:
     "Boards": ""
     "Conferences": ""
     "Cores": ""
-    "Cores:Collection": "collections containing multiple IP cores"
-    "Cores:Library": "libraries containing multiple IP cores"
-    "Cores:Single": "single IP core"
+    "Cores:Collection": "Groups of individual IP cores"
+    "Cores:Library": "Strong-related set of IP cores"
+    "Cores:Single": "Single IP cores"
     "Frameworks": ""
-    "Frameworks:Verification": "frameworks for verification"
+    "Frameworks:Verification": "Frameworks for verification"
+    "Languages": ""
     "Resources": ""
-    "Resources:Books": ""
-    "Resources:Social Media": ""
-    "Resources:Websites": ""
-    "Resources:Weekly": ""
-    "Resources:Wiki": ""
+    "Resources:Books": "Books"
+    "Resources:Social Media": "Social Media Network pages"
+    "Resources:Websites": "Websites"
+    "Resources:Weekly": "Weekly publications"
+    "Resources:Wiki": "Wikis"
     "Services": ""
-    "Services:Code Quality": ""
-    "Services:Continuous Documentation": ""
-    "Services:Continuous Integration": ""
+    "Services:Code Quality": "Code Quality services"
+    "Services:Continuous Documentation": "Continuous Documentation services"
+    "Services:Continuous Integration": "Continuous Integration services"
     "Tools": ""
-    "Tools:Bitgen": ""
-    "Tools:Grammars": ""
+    "Tools:Bitgen": "Bitstream generation programs"
+    "Tools:Grammars": "Grammars"
     "Tools:Managers": ""
     "Tools:Package Managers": ""
     "Tools:Parsers": ""
-    "Tools:PnRs": ""
-    "Tools:Programmers": ""
-    "Tools:Simulators": ""
-    "Tools:Synthesizers": ""
-    "Tools:Verification": ""
-    "Tools:Waveform Viewer": ""
+    "Tools:PnRs": "Place and Route programs"
+    "Tools:Programmers": "Programmers"
+    "Tools:Simulators": "Simulators"
+    "Tools:Synthesizers": "Synthesizers"
+    "Tools:Verification": "Verification"
```

Comment:
I am not sure what to include here, having "Frameworks:Verification: and "Verification".

Reply:
Tools belong here, while libraries and scripts correspond to Verification or Frameworks:Verification. For example, Symbiyosys is a tool for formal verification, but it's useless without other tools for synthesis. A framework would include a synthesis tool, symbiyosys and helper libraries to be used all together.